_pages/security.md

```diff
@@ -3,9 +3,9 @@ title: General Security Standards
 navtitle: Security
 ---
 
-In the Federal government, the principal law governing the security of information systems is the Federal Information Security Management Act (FISMA). For more information on FISMA, check out the [FISMA Ready introduction](https://github.com/fisma-ready/introduction), a community project co-managed by TTS. Cloud.gov and the [TTS Tech Portfolio](https://handbook.18f.gov/tech-portfolio/) team have done most of the heavy lifting with regards to FISMA, but it is still important to understand the context into which we ship.
+In the Federal government, the principal law governing the security of information systems is the Federal Information Security Modernization Act (FISMA). For more information on FISMA, check out the [FISMA Implementation Project ](https://csrc.nist.gov/projects/risk-management), which will help you stay up to date and in the know about all things FISMA. If you're inclined to see TTS' intepretation of FISMA from a user-center approach, take a look at a previous project from the TTS Tech Portfolio and cloud.gov known as [FISMA Ready](https://github.com/fisma-ready/introduction).  
```

Comment:
I didn't realize there were two!

> The Federal Information Security Modernization Act of 2014 amends the Federal Information Security Management Act of 2002

https://csrc.nist.gov/projects/risk-management/detailed-overview

Reply:
Yup it was a 🤯  when I first realized it too